osimpipeline/vital_tasks.py

```diff
@@ -1,15 +1,21 @@
 
 import os
+import time
 import task
 from doit.action import CmdAction
 from numpy import loadtxt
 import shutil
 import pylab as pl
+import pandas as pd
+import numpy as np
+import scipy.io as sio
+from scipy.interpolate import interp1d
+import h5py
```

Comment:
Does this actually need to be imported in this file?

Reply:
Oh, I guess not. It's probably a leftover before I created the HDF conversion utility functions.